src/notes/edit.ts

```diff
@@ -28,7 +30,9 @@ export default (content: HTMLElement, tabId: string): void => {
     modifiedTime: new Date().toISOString(),
   };
   localStorage.setItem("notesToSave", JSON.stringify(notesToSave));
+  clearTimeout(notesChangedByTimeout);
   localStorage.setItem("notesChangedBy", tabId);
+  notesChangedByTimeout = window.setTimeout(() => localStorage.removeItem("notesChangedBy"), 2000);
```

Comment:
It is important to reset "notesChangedBy" otherwise editing clipboard manually and from the background (using Context menu to save the selected text) and vice-versa, would NOT be enough to know if the change was done by the user or from the background, where if done from the background, clipboard note content should be automatically refreshed.